templates/settings-admin.php

```diff
@@ -55,4 +55,9 @@
 			   value="1" <?php if ($_['enforceSpecialCharacters']) print_unescaped('checked="checked"'); ?> />
 		<label for="password-policy-enforce-special-characters"><?php p($l->t('Enforce special characters'));?></label><br/>
 	</p>
+	<p id="enforceHaveIBeenPwned">
+		<input type="checkbox" name="password-policy-enforce-have-i-been-pwned" id="password-policy-enforce-have-i-been-pwned" class="checkbox"
+			   value="1" <?php if ($_['enforceHaveIBeenPwned']) print_unescaped('checked="checked"'); ?> />
+		<label for="password-policy-enforce-have-i-been-pwned"><?php p($l->t('Enforce checking against the list of breached passwords from haveibeenpwnd.com'));?></label><br/>
```

Comment:
Check password against the list of breached passwords from haveibeenpwnd.com

Reply:
Add a little text:

```html
<p class="password-policy-settings-hint">
This check creates a hash of the password and sends the first 5 characters of this hash to the haveibeenpwnd.com API to retrieve a list of all hashes that start with those. Then it checks on the Nextcloud instance if the password hash is in the result set.
</p>
```

Then create following style (there is no suitable in the server styles):

```css
.password-policy-settings-hint {
    opacity: 0.7;
    padding-left: 19px;
}
```